doc/lua_api.md

```diff
@@ -268,7 +268,7 @@ Media files (textures, sounds, whatever) that will be transferred to the
 client and will be available for use by the mod and translation files for
 the clients (see [Translations]). Accepted characters for names are:
 
-    a-zA-Z0-9_.-
+    a-z,A-Z,0-9 and _.-
```

Comment:
In my opinion, the previous one is easier to understand.
The commas are just distracting, and it would be more correct to write `a-z,A-Z,0-9,_,. and -`